testing/samurai/APKBUILD

```diff
@@ -0,0 +1,27 @@
+# Contributor: TBK <alpine@jjtc.eu>
+# Maintainer: TBK <alpine@jjtc.eu>
+pkgname=samurai
+pkgver=0.3
+pkgrel=0
+pkgdesc="Ninja-compatible build tool written in C99 with focus on simplicity, speed and portability."
+url="https://github.com/michaelforney/samurai"
+arch="all"
+license="Apache-2.0 MIT"
+options="!check" # No test suite
+provides="ninja"
```

Comment:
You probably don't want this as it will add a second package to aports that provides ninja and i bet that the original ninja package has a higher version number. When apk finds multiple provides it will choose the one with the highest version number. Instead change it to replaces so you are able to install this instead of original ninja.

Reply:
Thanks for the info. I have changed it to `replaces="ninja"`.